text/3-entrees/fixed-point.md

```diff
@@ -0,0 +1,329 @@
+# Fixed Point Arithmetic
+*[`pallets/fixed-point`](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/fixed-point)* *[`pallets/compounding-interest`](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/compounding-interest)*
```

Comment:
Is this line normally here?

Reply:
Yes. I've been placing a link from the text to the code it describes in each write-up. Here are a few [other](https://github.com/substrate-developer-hub/recipes/blame/master/text/3-entrees/basic-pow.md#L2) [examples](https://github.com/substrate-developer-hub/recipes/blame/master/text/3-entrees/constants.md#L2).

This one is slightly unique because this write-up is describing two pallets rather than one.